libr/core/cio.c

```diff
@@ -106,35 +106,36 @@ R_API bool r_core_dump(RCore *core, const char *file, ut64 addr, ut64 size, int
 	return true;
 }
 
-static inline void __endian_swap(ut8 *buf, ut32 blocksize, ut8 len) {
+static int __endian_swap(ut8 *buf, ut32 blocksize, ut8 len) {
```

Comment:
Notice I modified the return type to notify the caller when the parameters are not valid. Then accordingly in line [264](https://github.com/radare/radare2/pull/14789/commits/e5f26cf914307a615fefa410e3cf9c08fa2b291b#diff-ca04423ab87af7e69e9adc8535309fe7R264) it does not process in case of error.

Reply:
Return bool, not int